openedx/features/course_experience/templates/course_experience/course-outline-fragment-new.html

```diff
@@ -26,6 +26,9 @@
                             id="${ section['id'] }">
                         <span class="fa fa-chevron-right" aria-hidden="true"></span>
                         <h3 class="section-title">${ section['display_name'] }</h3>
+                        % if section.get('complete') and show_visual_progress:
```

Comment:
#NIT I like the `show_visual_progress` flag being checked first. It reads better in my opinion and performs faster (infinitesimally faster)